src/set_theory/cardinal/basic.lean

```diff
@@ -565,27 +565,13 @@ cardinal.wf.conditionally_complete_linear_order_with_bot 0 $ le_antisymm (cardin
 
 instance wo : @is_well_order cardinal.{u} (<) := ⟨cardinal.wf⟩
 
-/-- The set in the definition of `cardinal.succ` is nonempty. -/
+/-- The set in the definition of `order.succ` for cardinals is nonempty. -/
 theorem succ_nonempty (c : cardinal) : {c' : cardinal | c < c'}.nonempty := ⟨_, cantor c⟩
```

Comment:
Is this a `no_top_order` instance in disguise?

Reply:
You're right, this can be proven by `exists_gt c`. I'll get rid of this theorem.